### ISSUE TITLE
Ticket Sales Cheat Sheet: Localise communication cheat sheet

### ISSUE THREAD

**Issue by KGorr** [mozilla/mozillafestival.org#973]
We are requesting that the communication cheat sheet be localised in 5 languages (French, Spanish, German, Hindu, Swahili).

FInal cheat sheet copy here: https://docs.google.com/document/d/1jTn78OyXn0SiMrz26CWXzeh-NbQQ7aiFAwV-GhuWFtA/edit?usp=sharing 

/cc @Saallen

**Comment by Saallen** [comment]
@TheoChevalier putting this back at the top of your github notifications

**Comment by TheoChevalier** [comment]
Hey, sorry for the lack of update here, but everything has been exposed on Thursday last week and is ongoing!

Status: https://l10n.mozilla-community.org/langchecker/?locale=all&website=6&file=other/2018/mozfest_tickets.lang
Right now, de, fr, pt-BR are fully ready! For the other locales, I’ve sent an update to everyone who participate in the first file

Translations: https://l10n.mozilla-community.org/langchecker/?website=6&file=other/2018/mozfest_tickets.lang&action=translate&show 

Side note: I’ve noticed a tiny typo in English while translating, “sneak peaks” vs. “sneak peeks”.

**Comment by sabrinang** [comment]
The status looks like 3 locales are still in progress, moving to this heartbeat

**Comment by sabrinang** [comment]
@TheoChevalier This looks like it is still in progress, moving to this heartbeat

**Comment by sabrinang** [comment]
@KGorr @TheoChevalier 
Can you both connect and see if we can wrap this issue up? Sarah suggested we should share what we have so far.

Moving to this heartbeat.

**Comment by KGorr** [comment]
Will do and will post an update soon.

**Comment by KGorr** [comment]
Update: I'll be publishing the finished translations today. The other Spanish should be done soon (waiting on the team return from holiday).

**Comment by KGorr** [comment]
Updated translations added here: https://docs.google.com/document/d/1jTn78OyXn0SiMrz26CWXzeh-NbQQ7aiFAwV-GhuWFtA/edit#

**Comment by gideonthomas** [comment]
Hi @KGorr, are we only waiting on Swahili translations?

**Comment by KGorr** [comment]
@gideonthomas - correct!

**Comment by gideonthomas** [comment]
awesome, thanks! Do we have a deadline for this? And I assume you will coordinate with Theo to get those translations?

**Comment by TheoChevalier** [comment]
Yes, also taking this opportunity to clarify that these are good to have, and higher priority requests can (and are) stealing team’s focus, so we shouldn’t block on this and keep publishing what’s ready.

**Comment by KGorr** [comment]
@gideonthomas - yes, I am working with @TheoChevalier. And all current translations are live on the cheat sheet, so no blockers.  If we are able to get the remaining translations, the only deadline is the week of the festival. Thanks all!
